plugins/notifications-backend/src/service/types.ts

```diff
@@ -30,9 +30,16 @@ export type Notification = {
   actions: NotificationAction[];
 };
 
+// Keep in sync with FE: plugins/notifications-frontend/src/api/notificationsApi.ts
 export type NotificationsFilter = {
   containsText?: string;
   createdAfter?: Date;
   messageScope?: 'all' | 'user' | 'system';
   user?: string;
 };
+
+// Keep in sync with FE: plugins/notifications-frontend/src/api/notificationsApi.ts
+export type NotificationsQuerySorting = {
+  fieldName: 'title' | 'message' | 'created' | 'origin' | 'topic';
+  direction: 'asc' | 'desc';
```

Comment:
same here. we get the value via request. no one validates it for us. so we end up validating the supported values somewhere else. so why put it here?

Reply:
If the furher processing on BE would be more complex, it's beneficial to have the type throughout various flows - if nothing else then as a reesurance we did not mixed it up somewhere.

But I agree, this is not our case - we pass it to the DB almost right after checking the value.